exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusUnitsHelper.java

```diff
@@ -5,208 +5,84 @@
 
 package io.opentelemetry.exporter.prometheus;
 
-import static io.opentelemetry.exporter.prometheus.NameSanitizer.SANITIZE_CONSECUTIVE_UNDERSCORES;
+import io.prometheus.metrics.model.snapshots.Unit;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.annotation.Nullable;
 
-import io.opentelemetry.api.internal.StringUtils;
-import java.util.regex.Pattern;
+/** Convert OpenTelemetry unit names to Prometheus units. */
+class PrometheusUnitsHelper {
 
-/**
- * A utility class that contains helper function(s) to aid conversion from OTLP to Prometheus units.
- *
- * @see <a
- *     href="https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#units-and-base-units">OpenMetrics
- *     specification for units</a>
- * @see <a href="https://prometheus.io/docs/practices/naming/#base-units">Prometheus best practices
- *     for units</a>
- */
-final class PrometheusUnitsHelper {
```

Comment:
@psx95 would you care to review the changes to this file to see if any of the work in #5400 is undone?

Reply:
Apologies for the delayed response @jack-berg , this notification got missed - I will take a look once and raise any concerns I find.

Reply:
Thanks @psx95!